Static.xcodeproj/project.pbxproj

```diff
@@ -564,7 +580,8 @@
 		21826AC21B3F51A100AA9641 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				INFOPLIST_FILE = Static/Tests/Info.plist;
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
```

Comment:
Would you mind elaborating on this change? Is it required for spm package support? The documentation says
> Always embed the Swift standard libraries in the target's products, even if the target does not contain any Swift code. For example, this should be enabled if the target is embedding other products which contain Swift, or if it is a test target which does not contain Swift but which is testing a product which does. This setting only applies to wrapped products, not to standalone binary products.

Reply:
Xcode did this automatically